velox/exec/RowContainer.cpp

```diff
@@ -634,7 +634,7 @@ int64_t RowContainer::sizeIncrement(
       AllocationPool::kMinPages * memory::AllocationTraits::kPageSize;
   int32_t needRows = std::max<int64_t>(0, numRows - numFreeRows_);
   int64_t needBytes =
-      std::min<int64_t>(0, variableLengthBytes - stringAllocator_.freeSpace());
+      std::max<int64_t>(0, variableLengthBytes - stringAllocator_.freeSpace());
```

Comment:
Is this a bug fix for an unrelated issue? It would be nice to mention this in the PR description.